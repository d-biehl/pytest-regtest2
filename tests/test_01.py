# encoding: utf-8
from __future__ import print_function, division, absolute_import


def test_should_allways_fail(regtest, tmpdir):
    print("sdlfkjadflkadjsf ", file=regtest)
    print(tmpdir.strpath, file=regtest)
    print("sdlfkjadflkadjsf ", file=regtest)


def test_bytes(regtest, tmpdir):
    contains_bin = "".join(chr(i) for i in range(13, 32))
    print(contains_bin, file=regtest)
