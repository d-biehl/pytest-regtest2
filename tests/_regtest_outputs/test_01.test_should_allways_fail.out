sdlfkjadflkadjsf
/private/var/folders/k8/zfp7dvcs1m326gz1brql1tv80000gn/T/pytest-of-uweschmitt/pytest-647/test_should_allways_fail0
sdlfkjadflkadjsf
